stable/elasticsearch-exporter/templates/service.yaml

```diff
@@ -7,14 +7,19 @@ metadata:
     app: {{ template "elasticsearch-exporter.name" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+    {{- if .Values.service.labels }}
+    {{- range $key, $value := .Values.service.labels }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+    {{- end }}
 {{- if .Values.service.annotations }}
   annotations:
 {{ toYaml .Values.service.annotations | indent 4 }}
 {{- end }}
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - name: http
+    - name: metrics
```

Comment:
Isn't this a breaking change with mechanisms depending on port name, like prometheus operator's ServiceMonitor?

Reply:
@desaintmartin Yes, you're right. I just added a new property to remain backwards compatible.

Reply:
@desaintmartin Let me know what you think